images/access-setup/content/bin/setup_compute.sh

```diff
@@ -176,6 +197,32 @@ tekton_chains_manifest(){
   printf "OK\n"
 }
 
+tekton_results_manifest(){
+  manifest="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
+  if [ ! -e "$manifest" ]; then
+    manifests_dir="$(dirname "$manifest")"
+    mkdir -p "$manifests_dir"
+    if [[ -z $TEKTON_RESULTS_DATABASE_USER || -z $TEKTON_RESULTS_DATABASE_PASSWORD ]]; then
+      printf "[ERROR] Tekton results database variable is not set, either set the variables using \n \
+      the config.yaml under tekton_results_db \n \
+      Or create '%s' \n" "$manifest" >&2
+      exit 1
+    fi
+
+    {
+      echo "---"
+      kubectl create namespace openshift-pipelines --dry-run=client -o yaml
+      echo "---"
+      kubectl create secret generic -n openshift-pipelines tekton-results-database --from-literal=DATABASE_USER="$TEKTON_RESULTS_DATABASE_USER" --from-literal=DATABASE_PASSWORD="$TEKTON_RESULTS_DATABASE_PASSWORD" --dry-run=client -o yaml
+    } > "$manifest"
+    if [ "$(yq ".data" < "$manifest" | grep -cE "^*DATABASE_USER:|^*DATABASE_PASSWORD:")" != "2" ]; then
```

Comment:
no need for wildcards (they are producing warnings) just use `grep -cE "DATABASE_USER|DATABASE_PASSWORD"` it should work fine

Reply:
updated now.